pkg/odo/cli/utils/config.go

```diff
@@ -53,12 +54,14 @@ var configurationSetCmd = &cobra.Command{
 Available Parameters:
 UpdateNotification - Controls if an update notification is shown or not (true or false)
 NamePrefix - Default prefix is the current directory name. Use this value to set a default name prefix.
-Timeout - Timeout (in seconds) for OpenShift server connection check`,
+Timeout - Timeout (in seconds) for OpenShift server connection check
+PodTimeout - Pod Timeout (in seconds) to wait for an OpenShift Pod to start`,
```

Comment:
it looks a bit weird that we have in two places, could we put a description of Available Parameters into one place?